test/public-api/me.e2e-spec.ts

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Please split renaming this file into a separate commit

Reply:
done